homeassistant/components/reolink/host.py

```diff
@@ -68,8 +68,6 @@ def api(self):
 
     async def async_init(self) -> None:
         """Connect to Reolink host."""
-        self._api.expire_session()
```

Comment:
Is this change related to this PR?

Reply:
No, sorry, I have reverted that change.
I will make a seperate PR to remove that.